packages/smooth_app/lib/database/dao_string_list_map.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:hive_flutter/hive_flutter.dart';
+import 'package:smooth_app/database/abstract_dao.dart';
+import 'package:smooth_app/database/local_database.dart';
+
+class DaoStringListMap extends AbstractDao {
+  DaoStringListMap(final LocalDatabase localDatabase) : super(localDatabase);
+
+  static const String _hiveBoxName = 'robotoffMap';
+  static const String _key = 'votedHistory';
```

Comment:
Of course that will have to be changed when we have another use for `DaoStringListMap`, but for the moment that's OK.